include/seastar/core/circular_buffer.hh

```diff
@@ -194,9 +194,10 @@ private:
        difference_type operator-(const cbiterator<CB, ValueType>& rhs) const noexcept {
             return idx - rhs.idx;
         }
+        cbiterator() = default;
     private:
-        CB* cb;
-        size_t idx;
+        CB* cb{nullptr};
+        size_t idx{0};
```

Comment:
These are unnecessary (default constructed iterator is undefined), but I guess doesn't hurt.